src/build/app/mod.rs

```diff
@@ -386,6 +432,86 @@ impl<'b> App<'b> {
         self
     }
 
+    /// Allows the subcommand to be used as if it were an [`Arg::short`]
+    ///
+    /// Sets the short version of the subcommand flag without the preceeding `-`.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use clap::{App, Arg};
+    /// App::new("pacman").subcommand(
+    ///    App::new("sync")
+    ///        .short_flag('S'),
+    /// )
+    /// # ;
+    /// ```
```

Comment:
This example is useless. Either extend it so it shows actual usage and result, or just remove. The example below is fine. Maybe add a link to the standalone example?